tests/sentry/nodestore/bigtable/backend/tests.py

```diff
@@ -10,7 +10,7 @@
 @pytest.mark.skip(reason="Bigtable is not available in CI")
 class BigtableNodeStorageTest(TestCase):
     def setUp(self):
-        self.ns = BigtableNodeStorage()
+        self.ns = BigtableNodeStorage(project="test")
```

Comment:
The project must be passed to the client to run these tests. Not sure if that's a new requirement, but it was not happy without it.